bigtable-hbase-parent/bigtable-hbase-mapreduce/pom.xml

```diff
@@ -137,6 +137,10 @@ limitations under the License.
                     <groupId>org.slf4j</groupId>
                     <artifactId>slf4j-api</artifactId>
                 </exclusion>
+                <exclusion>
```

Comment:
interesting.  I guess we'll have to add the comment everywhere we do this.  I guess it really is only eclipse.